.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,10 @@
+name: lint_python
+on: [push]
```

Comment:
Would you like to run it on pull request as well?

Reply:
Everyone pushes but _insiders_ do not pull request.